app/models/good_job/execution.rb

```diff
@@ -150,9 +150,9 @@ def self.queue_parser(string)
     # @return [ActiveRecord::Relation]
     scope :schedule_ordered, -> { order(coalesce_scheduled_at_created_at.asc) }
 
-    # Get Jobs were completed before the given timestamp. If no timestamp is
-    # provided, get all jobs that have been completed. By default, GoodJob
-    # destroys jobs after they are completed and this will find no jobs.
+    # Get completed jobs before the given timestamp. If no timestamp is
+    # provided, get *all* completed jobs. By default, GoodJob
+    # destroys jobs after they're completed, meaning this returns no jobs.
```

Comment:
Can back this out, but this was a little funky. Do you use capital-j Job a lot in comments to refer to Job instances? Could these usages be lowercase j generic "jobs"?

Reply:
this is ok 👍🏻 Reading through this, I think there's some historical baggage here given this is in `GoodJob::Execution` but referring to jobs. I think the language is an improvement.